include/tilck/kernel/modules.h

```diff
@@ -34,4 +34,5 @@ void register_module(struct module *m);
 #define MOD_fbdev_prio                       300
 #define MOD_serial_prio                      400
 #define MOD_sb16_prio                        410
-#define MOD_dp_prio                         1000  /* last */
+#define MOD_dp_prio                         1000
+#define MOD_systests_prio                   1010  /* last */
```

Comment:
Oh, ok. I didn't had any priority number in mind so I just placed it last (which I need to correct). Also, is there any kind of rule which I need to follow to select the priority number?

Reply:
@0xdeadbad Yes, there are dependencies. The "dp" (debug panel) module needs to be initialized last, so that it could see the status of all the other modules. If the new module doesn't depend on anything, it could be placed anywhere, but I guess it's better to put it just immediately before "dp" just in case in the future it needs to access data structures initialized by the other modules.